data_model.md

```diff
@@ -349,14 +349,19 @@ timestamp,value,quality_flag
 ### Uploads
 {: .anchor}
 
-Data may be uploaded to Solar Forecast Arbiter either through the API or
+Data may be uploaded to the Solar Forecast Arbiter either through the API or
 the dashboard in either CSV or JSON format. Valid files must have
 timestamps in ISO-8601 format with time zone. They must also include the
-value of the data, and a quality flag of either 0 or 1. Each file
-may only contain data for **one** observation or forecast variable.
+value of the data, and a quality flag of either 0 or 1.
+
+Each file may only contain data for **one** observation or forecast variable.
```

Comment:
I expect we'll get pushback on this restriction, apologies for raising this comment again.

Reply:
I agree but we've not seen any pushback on this in the limited feedback so far. We're still thinking that we'll add some kind of support for multiple observations per file after the June alpha release.